AdminCore.WebApp/Controllers/EventController.cs

```diff
@@ -115,6 +119,16 @@ public IActionResult GetEventByStatusType(int eventStatusId, int eventTypeId)
       return StatusCode((int)HttpStatusCode.NoContent, "No Event exists");
     }
 
+    [HttpGet("findByEventStatus/{eventStatusId}/{eventTypeId}/csv")]
+    public IActionResult GetEventByStatusTypeCsv(int eventStatusId, int eventTypeId)
+    {
+      var events = _eventService.GetEventByStatus((EventStatuses)eventStatusId, (EventTypes)eventTypeId);
+      var mappedEvents = _mapper.Map<IList<EventDataTransformModel>>(events);
+
+      var stream = new MemoryStream(_csvService.Generate(mappedEvents));
+      return File(stream, "application/octet-stream", "Report.csv");
```

Comment:
I'd include the date in the name.

Reply:
can we figure out a way to make report names meaningful?

Reply:
I've pushed a commit that now gives them names such as "AwaitingApproval_AnnualLeave_Report13-05-2019.csv"

Reply:
that's perfect, thanks :+1: